assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+* Teams owning the repository are not complying with mandates for having a CONTRIBUTING.md
+* Compliance guys have to do a job; this is made difficult by teams resisting this.
+* Teams are focusing on business value; CONTRIBUTING.md compliance is viewed as not as important.
+* Special task force for security and compliance: developers responsible for fixing these bugs across the company.
+* You have to do these fixes.
+* Negotiations are being held but they are slow and little progress is made.
+* The task force is discovering the needed information for a CONTRIBUTING.md as they investigate the problem.
+* There is a checklist for being InnerSource compliant.
+* There is a need to be Security compliant.
+* There may be export control compliance and legal compliance requirements; a template is provided to repository owners
+
+## Forces
+* Teams have been resisting this; this ends up wasting time.
+* Making documents part of the repo skeleton might be "rubber stamping"; better to have teams own this. So too much automation in this case is bad.
+
+## Solution
+* Rather than asking the resisting team to do the changes, they create the documentation (in addition to negotiations)
```

Comment:
```suggestion
* Rather than asking the resisting team to do the changes, the policing task force that discovers the lack of compliance create the documentation (in addition to negotiations)
```

Fixing as suggested in https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/74/files#r169645540